.swiftpm/xcode/xcshareddata/xcschemes/Apollo.xcscheme

```diff
@@ -1,113 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's odd that these are deleted but note that this is the `.swiftpm` directory. I checked the apollo-ios Xcode project, in this branch, and the schemes are still there and shared so it must just be an Xcode SPM project thing.